modules/redis/redis.go

```diff
@@ -54,7 +54,7 @@ func Run(ctx context.Context, img string, opts ...testcontainers.ContainerCustom
 	req := testcontainers.ContainerRequest{
 		Image:        img,
 		ExposedPorts: []string{"6379/tcp"},
-		WaitingFor:   wait.ForLog("* Ready to accept connections"),
+		WaitingFor:   wait.ForListeningPort("6379/tcp"),
```

Comment:
I haven't seen any issue with this wait strategy, which is widely use here and in the java module.

Could you bring more light about the motivation for the change?

Reply:
With all the iterations on the test suite I've had a few cases where test failed due to redis not accept incoming requests.

I didn't dig in fully as with this change, I never saw a reoccurrence.  It could be redis reports the log entry early?

Reply:
I'd keep this wait strategy as is, as it's been used for years. The PR is so simple that we can go back to it whenever it's needed. In https://github.com/testcontainers/testcontainers-go/issues/2633 we added a wait for all available ports, but it contains a bug, reported in https://github.com/testcontainers/testcontainers-go/issues/2633

I'd close this one because of that

Reply:
Sure will keep an eye for a failure.